src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java

```diff
@@ -799,6 +800,10 @@ public StampType getStampType(FinremCaseData caseData) {
         return isHighCourtSelected(caseData) ? StampType.HIGH_COURT_STAMP : StampType.FAMILY_COURT_STAMP;
     }
 
+    public LocalDate getConsentDateOfOrder(Map<String, Object> caseData) {
```

Comment:
This doesn't feel like the appropriate place to put this function and it's also inconsistent with how you've retrieved the date in other places e.g.
`LocalDate approvalDate = caseData.getConsentOrderWrapper().getConsentDateOfOrder();`

Do you really need this method here?